### PR TITLE
Remove nf-core modules check

### PR DESCRIPTION
**Pull request by ewels**
- [x] Remove "check" command
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615806091239300
- [x] Make it more obvious that commands are not yet implemented in help
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615805322230000
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615805565234000
- [x] Module should not be "install" or "remove" if pipeline directory is invalid
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615804113221800
  https://nfcore.slack.com/archives/CE5LG7WMB/p1615804546224400
- [x] Remove "def options" from the modules template
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615803595218500
- [x] Rename "modules/.github/filters.yml" -> "tests/pytest_include.yml"
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615807094244800
- [x] List modules installed in pipeline as well as on nf-core/modules 
    * https://nfcore.slack.com/archives/CE5LG7WMB/p1615803595218500

### CODE REVIEW

File: nf_core/modules/create.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def create(self):
         modules/tests/software/tool/subtool/
             * main.nf
             * test.yml
-        modules/.github/filters.yml
+        modules/tests/pytest_include.yml
```
Comment (drpatelh) [inline nf_core/modules/create.py:62]:
```suggestion
        tests/config/pytest_software.yml
```

Comment (drpatelh) [reply nf_core/modules/create.py:62]: As changed in https://github.com/nf-core/tools/pull/879

**Comment by drpatelh** [comment]
Do we remove the `update` command too?

**Comment by ewels** [comment]
Made an awesome fuzzy-finder interface when installing modules which I kind of love:

Mar-15-2021 15-12-09

**Comment by ewels** [comment]
> Do we remove the `update` command too?

Yup, it's gone from the command line interface. I commented it out and left the stub as we do intend to put that in at a later date (I completely deleted `check` as that's part of `lint`).

**Comment by drpatelh** [comment]
As mentioned on Slack we need to change where these modules are installed locally:
https://github.com/nf-core/tools/blob/d0b7daff242ac33a2357c2ffa54bd6e982be5725/nf_core/modules/create.py#L51

Need to change this install path to `directory/modules/local/` i.e. without the `process` - based on recent module restructuring in #viralrecon

**Comment by ewels** [comment]
Dropped `process` dir in 336de7d 👍🏻

**Review by drpatelh — APPROVED** [review]
Pleasure as always Sir! 💥

**Pull request merged**